apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Playground/Chat/index.tsx

```diff
@@ -110,9 +110,8 @@ export default function Chat({
 
       switch (event) {
         case StreamEventTypes.Latitude: {
-          if (data.type === ChainEventTypes.Step) {
-            if (data.isLastStep) setChainLength(messagesCount + 1)
-          } else if (data.type === ChainEventTypes.Complete) {
```

Comment:
This was a bug. We lost running time line because `promptl` does not know what `isLastStep` so we can't rely on that. We do this instead of chain complete.
![image](https://github.com/user-attachments/assets/f94eddf3-a61d-4ead-a081-c25cc4560f0d)